src/lib/filters/status.js

```diff
@@ -5,6 +5,8 @@ export const status = {
   choices: [
     { value: 'open', label: 'Open' },
     { value: 'closed', label: 'Closed' },
+    { value: 'merged', label: 'Merged' },
+    { value: 'unmerged', label: 'Unmerged' },
```

Comment:
`is:unmerged` and `is:closed/open` aren't mutually exclusive.

A PR can be `unmerged & closed` or `unmerged & open`.

I think we should have a new predicate for this, called `Merge status` maybe?

---

This can be useful if you want to see all the PRs that are `closed` but not `merged` for instance